starter-code/css/main.css

```diff
@@ -6,3 +6,187 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+  font-family: 'Open Sans', sans-serif;
+  margin: 0px auto;
+}
+
+/* text styles */
+
+h1 {
+font-size: 82px;
+color: white;
+font-weight: bold;
+padding-top: 250px;
+}
+
+h2 {
+font-size: 36px;
+color: white;
+font-weight: 100;
+padding-top: 25px;
+}
+
+h3 {
+  
+}
+
+h4 {
+  
+}
+
+p {
+  color:#1A1A1A;
+}
+
+li {
+  float: right;
+  list-style-type: none;
+}
+
+a {
+  display: block;
+  padding: 8px;
+}
+
+/* image styles */
+
+.logo {
+  max-width: 220px;
+  max-height: 70px;
+}
+
+/* structure styles */
+
+nav {
```

Comment:
Let's add class to "nav" next time so you don't get the habit to use tags since with Bootstrap it will get messy (you will overwrite the predefined Bootstrap classes if you keep doing this).